src/settings/url-settings.ts

```diff
@@ -0,0 +1,14 @@
+export const VALID_URL_PROTOCOLS = ["http:", "https:"];
```

Comment:
We are using `camelCase` for files and constants, please stick to the current convention.

Reply:
That's the original name https://github.com/sparkletown/sparkle/blob/staging/src/settings/index.ts#L157 so I guess it was already considered as a constant.

Reply:
Now made a constant 78400178ee91efd4c4cb778eb68c1980370fb3cc

Reply:
Thanks, can you please rename the file as well? The rest of the files in the project are also camelCase instead of `-` in between.